### DISCUSSION TITLE
Creating a team: personal account or business/institution?

### DISCUSSION THREAD

**Discussion by matthiashhh** [community/community#23813] [New to GitHub]
<p>I’m creating a team for an online community that wants to collaborate on coding projects. The code will be open source. Our community is informal: there is no registered organization, no leadership, no contact points (email or otherwise) other than a Slack workspace with an admin who is the initiator of the community. This person is not a programmer, and will therefore not be involved in github projects.</p>
<p>When creating the team, I get the following question:</p>
<pre><code>This organization belongs to:
- My personal account
- A business or institution
</code></pre>
<p>What would be the right choice here? As we are not a formal organization, I think it should be “My personal account”, but I don’t want to be seen by the community as hijacking some privileged role. What are the consequences of this choice, and are any of the consequences irrevertible?</p>

**Comment by Maxey012** [comment]
<p>Yes i think you might be go with the personel account because its the best option for all who are join your team the other<a href="http://sportszilla.org" rel="nofollow noopener">,</a> option is not suitable.</p>

**Comment by Maxey012** [comment]
<p>My personal account will be the best option fo for it</p>

**Comment by seveas** [comment, marked as answer]
<p>Personal account is the one to choose in your situation. For the business/institution option, you will be asked to agree to other terms of use, which e.g. include provisions for company representatives to assume control of the organization.</p>
<p>For ‘personal’ organizations, you’ll be responsible for this yourselves.</p>

**Reply by peoray** [reply]
can this be changed later on?
